R/article.R

```diff
@@ -489,11 +489,15 @@ sim_article <- function(..., highlight = NULL, citation_package = "natbib") {
 #'   Macro package for Springer Journals.
 #' @export
 #' @rdname article
-springer_article <- function(..., keep_tex = TRUE,
-                             citation_package = "default") {
+springer_article <- function(..., keep_tex = TRUE) {
+
+  if (!rmarkdown::pandoc_available("2.11.4")) {
+    stop("`springer_article()` now requires a minimum of pandoc 2.11.4")
+  }
+
   pdf_document_format(
     "springer",
-    keep_tex = keep_tex, citation_package = citation_package, ...
+    keep_tex = keep_tex, citation_package = "natbib", ...
```

Comment:
> Does this template only requires natbib for sure ?

No, I'm not sure, but the instructions in the template are tu use `\citet`. I can just use the default and assume that anyone willing to change it would understand what they're doing.